src/data/ScopeClosures/ch1.js

```diff
@@ -79,6 +79,181 @@ const Ch1Questions = [
   },
+  {
+    question: 'What is Parsing?',
+    orderedById: false,
+    answers: [
+      { text: 'Taking a string and converting it into tokens', id: 0 },
+      {
+        text:
+          'Taking a stream of tokens and converting them into an abstract syntax tree ',
+        id: 1,
+      },
+      {
+        text:
+          'Taking an abstract syntax tree and converting it into zeroes and ones',
+        id: 2,
+      },
+      {
+        text: 'Taking a string and converting it into assembly',
+        id: 3,
+      },
+    ],
+    correctAnswerId: 1,
+    moreInfoUrl:
+      'https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch1.md#compiler-theory',
+    explanation: `Parsing is typically thought of as taking a string of text and making sense of it. However, the string of text is actually converted into tokens first. Parsing is actually taking the tokens and making them useful by converting them into an abstract syntax tree`,
+  },
+  {
+    question: 'What is the responsibility of a JavaScript engine?',
+    orderedById: false,
+    answers: [
+      {
+        text: 'Handling all the dirty work of parsing and code-generation',
+        id: 0,
+      },
+      {
+        text: 'Writing JavaScript code that is missing from the file',
+        id: 1,
+      },
+      {
+        text:
+          'Start-to-finish compilation and execution of our JavaScript program',
+        id: 2,
+      },
+      {
+        text: 'Find all unused code and remove it before code generation',
+        id: 3,
+      },
+    ],
+    correctAnswerId: 2,
+    moreInfoUrl:
+      'https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch1.md#the-cast',
+    explanation: `The engine is the entirety of what reads and executes your code. Each browser has their own engine as does Node (built on Chrome's V8 JavaScript engine).`,
+  },
+  {
+    question: 'What is the difference between an LHS lookup and an RHS lookup?',
+    orderedById: false,
+    answers: [
+      {
+        text: 'LHS lookups search for variables whereas RHS search for values',
+        id: 0,
+      },
+      {
+        text:
+          'LHS looks for assignment operator whereas RHS looks for the value to be assigned',
+        id: 1,
+      },
+      {
+        text:
+          'LHS looks for the existence of a variable whereas RHS looks for the history of that variable',
+        id: 2,
+      },
+      {
+        text:
+          'LHS looks for the variable container whereas RHS looks for the value of the variable',
+        id: 3,
+      },
+    ],
+    correctAnswerId: 3,
+    moreInfoUrl:
+      'https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch1.md#compiler-speak',
+    explanation: `LHS stands for "left hand side", and RHS stands for "right hand side". For a = b, the RHS looks for the value of b, and the LHS looks for the location of a in memory, so that it can update its value.`,
```

Comment:
`the LHS looks for the location of a in memory`
had to read this a few times, maybe bold 'a' ?

Reply:
I could put `a` in quotes. We don't yet have the ability to style the questions.